src/uu/stty/src/stty.rs

```diff
@@ -120,7 +120,15 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 fn stty(opts: &Options) -> UResult<()> {
     // TODO: Figure out the right error message
     let mut termios = tcgetattr(opts.file).expect("Could not get terminal attributes");
+
     if let Some(settings) = &opts.settings {
+        if opts.save || opts.all {
```

Comment:
should be and and right ?? 
This is an or operator.

Reply:
Nope, the `||` is correct here I think. The error is about combining the `--save` flag or the `--all` with settings that you want to apply. Basically, `stty` will refuse to both print and change settings. I.e.:
```
❯ stty --save nl0
stty: when specifying an output style, modes may not be set 
❯ stty --all nl0
stty: when specifying an output style, modes may not be set
```
I did just realize that I can add a test for this if I restructure a bit, so I'm gonna do that.

Reply:
Oh but there is also a separate error for using both `--save` and `--all`, which I should add:
```
❯ stty --all --save
stty: the options for verbose and stty-readable output styles are
mutually exclusive
```

Reply:
Yes, the comment reminded me of that ^^'